pkg/controller/autoimport/autoimport_controller.go

```diff
@@ -133,6 +137,25 @@ func (r *ReconcileAutoImport) Reconcile(ctx context.Context, request reconcile.R
 		backupRestore = true
 	}
 
+	generateClientHolderFunc, err := r.getGenerateClientHolderFuncFromAutoImportSecret(managedClusterName, autoImportSecret)
+	if err != nil {
+		if err := helpers.UpdateManagedClusterStatus(
+			r.client,
+			managedClusterName,
+			helpers.NewManagedClusterImportSucceededCondition(
+				metav1.ConditionFalse,
+				constants.ConditionReasonManagedClusterImportFailed,
```

Comment:
I thought we already have an imported condition?

Reply:
currently, we use different condition to represent the cluster import process
https://github.com/stolostron/managedcluster-import-controller/blob/main/pkg/constants/constants.go#L108